app/controllers/quizmaster/quizzes_controller.rb

```diff
@@ -3,4 +3,8 @@ def show
     @quiz = Quiz.find(params[:id])
     @questions = @quiz.questions
   end
+
+  def broadcast_content
+    BroadcastQuizJob.perform_now('Whatup??')
+  end
```

Comment:
This is also unnecessary, right? Or are we going to put something useful in here next?